examples/v2/project_creation/project.py.schema

```diff
@@ -78,6 +78,13 @@ properties:
                 item:
                   type: string
                   pattern: "^(user|group|domain|serviceAccount):"
+  set-dm-service-account-as-owner:
+    description: >
+      Set true so that the service account that Deployment Manager will use
+      on the generated project is owner, so that it can set IAM policies
+      on resources created within that project.
+    type: boolean
+    default: true
```

Comment:
Can we default this to  false and explicitly set it to true in our sample yaml?

Reply:
Done